packages/admin-panel/src/VizBuilderApp/components/PreviewOptions.js

```diff
@@ -46,42 +47,126 @@ const PaperComponent = styled(MuiPaper)`
   }
 `;
 
-const LocationField = () => {
-  const [search, setSearch] = useState('');
-  const [{ project, location }, { setValue }] = useVizBuilderConfig();
+const ProjectField = ({ project, projects, isLoadingProjects, onChange }) => (
```

Comment:
Changes in this file is to add Project selection in Preview options:
![image](https://user-images.githubusercontent.com/63621318/128275975-bce1910d-3a0e-4ecd-80eb-d08f3e1f04f0.png)